manifests/init.pp

```diff
@@ -67,6 +67,58 @@
 #
 # $server_ca_name::       The name of the server CA (used for https)
 #
+# $repomd_gpg::                  Path to a directory containing a keyring to use
+#                                for repository metadata signing
+#                                If not specified, a GPG key will be generated
+#
+# $repomd_gpg_name::             Name attribute for the GPG key generated for
+#                                repository metadata signing
+#
+# $repomd_gpg_comment::          Comment attribute for the GPG key generated
```

Comment:
I'd suggest some of there should definitely be advanced options, or configurable via hiera.

Reply:
They are all advanced options.  I didn't add anything to the non-advanced parameter list.